daphne/server.py

```diff
@@ -38,6 +38,7 @@ class Server(object):
     def __init__(
         self,
         application,
+        lock,
```

Comment:
This is not serviceable; didn't realize i had strayed out of the testing code and into the actual public interface...